packages/react-native/ReactAndroid/src/main/java/com/facebook/react/common/LongArray.java

```diff
@@ -21,7 +21,7 @@ public static LongArray createWithInitialCapacity(int initialCapacity) {
 
   private LongArray(int initialCapacity) {
     mArray = new long[initialCapacity];
-    mLength = 0;
+    mLength = 2;
```

Comment:
**Suggestion:** The initial length of the array should be 0, not 2. This might cause an out of bounds exception if you try to access an index that hasn't been initialized yet.
```suggestion
  private LongArray(int initialCapacity) {
    mArray = new long[initialCapacity];
    mLength = 0;
  }
```